lib/text_sender.rb

```diff
@@ -0,0 +1,11 @@
+class TextSender
+
+  def order_confirmation twilio
```

Comment:
Unused method argument - `twilio`. If it's necessary, use `_` or `_twilio` as an argument name to indicate that it won't be used. You can also write as `order_confirmation(*)` if you want the method to accept any arguments but don't care about them.